doc/source/release/1.15.0-notes.rst

```diff
@@ -24,38 +24,151 @@ This release requires Python 3.10+ and NumPy 1.23.5 or greater.
 Highlights of this release
 **************************
 
+Sparse arrays are now fully functional for 1-D and 2-D arrays. We recommend
+that all new code use sparse arrays and that developers start to migrate their
+existing code from sparse matrix to sparse array:
+:ref:`migration_to_sparray`. Both ``sparse.linalg`` and ``sparse.csgraph``
+work with either sparse matrix or sparse array and work internally with
+sparse array.
```

Comment:
+1. I'd also suggest n-D COO arrays as a separate highlight.

Reply:
`scipy.differentiate` may also deserve a mention as a highlight? It's rare we add a new top-level submodule.

Reply:
I think @ollybritton plans to write something about `integrate.cubature` too, which can go in the highlights.

Reply:
I would also put forward the new `optimize.elementwise` methods and `interpolate.AAA`  for rational approximation. This looks like the best release in a while for new features!